README.md

```diff
@@ -11,6 +12,61 @@ You can reach the maintainers of this project at:
 - [Slack channel](https://kubernetes.slack.com/messages/sig-node)
 - [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-node)
 
+ 
+## Building the operator
+
+Checkout the sources
+
+```bash
+$ git clone https://github.com/kubernetes-sigs/node-feature-discovery-operator $GOPATH/src/github.com/kubernetes-sigs/node-feature-discovery-operator
+```
+
+Update the `Makefile` and edit `IMAGE` and `REGISTRY` one will need those later to update the operator manifest (`image: $REGISTRY/$IMAGE`).
+
+## Manual deploy of the operator
+
+Checkout the sources
+
+```bash
+$ git clone https://github.com/kubernetes-sigs/node-feature-discovery-operator
+```
+Update the  `Makefile` with the a custom image built and configure the namespace where the operator should be deployed.
```

Comment:
I would not duplicate these instructions here. I'd just refer to building manually (the section above).